demos/cms/cms_ddec.c

```diff
@@ -68,6 +68,9 @@ int main(int argc, char **argv)
     if (!CMS_decrypt(cms, rkey, rcert, dcont, out, 0))
         goto err;
 
+    printf("Successfully dencrypted contents of file smencr.pem"
```

Comment:
Please tell me what do you mean by "there probably need to be some newline characters here."? I am fixing it right now.